src/App.js

```diff
@@ -1083,7 +1090,7 @@ export const App = () => {
 									Container creation is a paid operation, make sure you have sufficient NeoFS chain balance
 								</Notification>
 							)}
-							<Heading className="input_caption">Container cost is 0.0070000014 GAS for mainnet now.</Heading>
+							<Heading className="input_caption">{`Container cost is ${networkInfo ? networkInfo.namedContainerFee * 0.00000001 : '-'} GAS for mainnet now.`}</Heading>
```

Comment:
It's _both_ `namedContainerFee` and `containerFee`. And it's in picoGAS, 10^-12.

Reply:
Ah, and it's also `×7`, paid per node.